unittesting-windows.md

```diff
@@ -0,0 +1,66 @@
+# Integrating WP Unit Testing in windows
+
+This document assumes that you have already tried setting up WP Unit Testing environment for your project in windows but have stucked somewhere.
+
+The purpose of this discussion is to perfectly set up WP Unit Testing environment in a local project. Followed through https://make.wordpress.org/cli/handbook/plugin-unit-tests/ but it lacks some points when it comes to windows with addition to some general points missing.
+
+First, some commands in wp-cli doesnt work with windows, so use cygwin or alike command line utility.
+Cygwin is preferred because its suggested even in <a href="/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=1&amp;cad=rja&amp;uact=8&amp;ved=0ahUKEwidhZ_RjqLVAhVBPZQKHXqPBUIQFgg1MAA&amp;url=https%3A%2F%2Fmake.wordpress.org%2Fcore%2Fhandbook%2Ftutorials%2Fworking-with-patches%2F&amp;usg=AFQjCNENVuRFv23KA3UugzN75VdTbklrdg" onmousedown="return rwt(this,'','','','1','AFQjCNENVuRFv23KA3UugzN75VdTbklrdg','','0ahUKEwidhZ_RjqLVAhVBPZQKHXqPBUIQFgg1MAA','','',event)" data-href="https://make.wordpress.org/core/handbook/tutorials/working-with-patches/">Working With Patches – Make WordPress Core</a>.
+
+Before starting, let me suggest that if you fail with running `wp scaffold` command then update your wp-cli to stable version first (Ex :  wp cli update --nightly)
+
+For ease of understanding, I rewrite the points of above mentioned tutorial, such as ...
+
+2) Generate the plugin test files:
```

Comment:
Ohh sorry, I mistaken ! There is no 1), the 2. relates second point of the original tutorial which I have mentioned right there in the second paragraph. So yes, there isn't the 1)